packages/optimizer/spec/transformers/valid/AmpBoilerplateTransformer/always_inlines_v0css/input.html

```diff
@@ -1,3 +1,9 @@
+<!--
+{
+  "ampRuntimeVersion": "0123456789",
```

Comment:
Since AMP has decided [RTVs must be 15-digits](https://github.com/ampproject/amphtml/issues/27579), we might want to get in the habit of using valid versions in examples and tests.